server/testserver.go

```diff
@@ -32,6 +35,7 @@ import (
 )
 
 // StartTestServer starts a in-memory test server.
+// Adds a permissions config for "test-user" under prefix "test-user".
```

Comment:
This is a very subtle difference between `StartTestServer()` and `(&TestServer{}).Start()`. I'd like to give `StartTestServer` a different name if these are going to diverge, but I don't have any better ideas.